packages/client/components/AddTeamDialog.tsx

```diff
@@ -51,57 +47,36 @@ const query = graphql`
 const AddTeamDialog = (props: Props) => {
   const {isOpen, onClose, queryRef, onAddTeam} = props
   const atmosphere = useAtmosphere()
+  const {history} = useRouter()
 
   const {submitting, onCompleted, onError, error, submitMutation} = useMutationProps()
-  const {history} = useRouter()
 
   const data = usePreloadedQuery<AddTeamDialogQuery>(query, queryRef)
   const viewer = useFragment<AddTeamDialog_viewer$key>(AddTeamDialogViewerFragment, data.viewer)
-  const {organizations: viewerOrganizations} = viewer
+  const {organization} = viewer
 
   const [selectedUsers, setSelectedUsers] = useState<Option[]>([])
-  const [mutualOrgsIds, setMutualOrgsIds] = useState<string[]>([])
+  const [isPublic, setIsPublic] = useState(true)
+  const [teamName, setTeamName] = useState('')
 
-  const showOrgPicker = !!(
-    selectedUsers.length &&
-    (mutualOrgsIds.length > 1 || !mutualOrgsIds.length) &&
-    viewerOrganizations.length > 1
-  )
+  if (!organization) return null
+  const isStarterTier = organization.tier === 'starter'
+  const disablePrivacyToggle = isStarterTier
 
-  const defaultOrgId = mutualOrgsIds[0]
-  const [selectedOrgId, setSelectedOrgId] = useState(defaultOrgId)
-  const [teamName, setTeamName] = useState('')
-  const [teamNameManuallyEdited, setTeamNameManuallyEdited] = useState(false)
-
-  const MAX_TEAM_NAME_LENGTH = 50
-  const generateTeamName = (newUsers: Option[]) => {
```

Comment:
The user can enter their team name. This led to confusing behaviour where I'd enter an email address to invite someone to the team and then the team name would be updated to the same name as the email